Gordon360/AuthorizationFilters/StateYourBusiness.cs

```diff
@@ -503,6 +550,32 @@ private bool canUpdate(string resource)
                     }
                 case Resource.STUDENT:
                     return false; // No one should be able to update a student through this API
+                case Resource.HOUSING:
+                    {
+                        // The housing admins can update the application information (i.e. probation, offcampus program, etc.)
+                        // If the user is a student, then the user must be on an application and be an editor to update the application
+                        var housingService = new HousingService(new UnitOfWork());
+                        if (user_position == Position.SUPERADMIN || (housingService.CheckIfHousingAdmin(user_id)))
```

Comment:
Again, you don't need to check for the superadmin case here because they are already authorized (see lines 61-72 of this file).